frontend/src/containers/ImageContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react';
+
+import ImageComponent from '../components/ImageComponent';
+import { IMAGE_API } from '../data/ApiConstants';
+import Image from '../models/Image';
+
+interface Props {}
+
+interface State {
+  images: Array<Image>;
+}
+
+class ImageContainer extends PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.state = { images: [] };
+  }
+
+  componentDidMount() {
+    fetch(IMAGE_API, { mode: 'cors' })
```

Comment:
one more tiny thing: `mode: 'cors'` is the default setting so you can probably just omit this second argument